src/main/resources/application.yml

```diff
@@ -13,7 +13,7 @@ spring:
     allow-bean-definition-overriding: true
   data:
     mongodb:
-      uri: mongodb://localhost:27017/chemicals
+      uri: mongodb://api-database/collection
```

Comment:
For docker, I think it's not needed to indicate /collection. In this solution, I used 2 Mongodb collections and they are hardcoded in the repository implementations.